frontend/src/components/widgets/DataFrame/styled-components.ts

```diff
@@ -51,6 +53,7 @@ export const StyledResizableContainer = styled.div<
 
   "& .dvn-scroller": {
     scrollbarWidth: "thin",
+    // https://caniuse.com/css-overflow-overlay only works on chrome and firefox
```

Comment:
So be clear, I think firefox this doesn't work based on the link.

My bet is that it doesn't work BUT the resizer cannot be hidden (a la the -webkit-resizer on line 62).